src/writeData/components/AddPluginToConfiguration.tsx

```diff
@@ -30,84 +41,121 @@ interface AddPluginToConfigurationCTAProps {
+                menu={onCollapse => (
+                  <Dropdown.Menu onCollapse={onCollapse}>
+                    <Dropdown.Item
+                      key="Create-new-configuration-telegraf-plugin"
+                      value="Create a new configuration"
+                      onClick={() => {
+                        history.push(
+                          `/${ORGS}/${orgID}/load-data/${TELEGRAF_PLUGINS}/${contentID}/new`
+                        )
+                      }}
+                      selected={false}
+                      testID="create-new-configuration-from-plugin--dropdown-item"
+                    >
+                      <span>Create a new configuration</span>
+                    </Dropdown.Item>
+                    {telegrafs?.length ? (
+                      <Dropdown.Item
+                        key="Add-to-existing-configuration-telegraf-plugin"
+                        value="Add to an existing configuration"
+                        onClick={() => {
+                          history.push(
+                            `/${ORGS}/${orgID}/load-data/${TELEGRAF_PLUGINS}/${contentID}/add`
+                          )
+                        }}
+                        selected={false}
+                      >
+                        <span>Add to an existing configuration</span>
+                      </Dropdown.Item>
+                    ) : null}
+                  </Dropdown.Menu>
+                )}
+                style={{width: 'auto'}}
+                testID="add-plugin-to-configuration--dropdown"
+              />
+            </Grid.Column>
+            <Grid.Column
+              widthXS={Columns.Twelve}
+              widthMD={Columns.Six}
+              widthLG={Columns.Nine}
             >
-              {pageContent}
-            </div>
-          </Grid.Column>
-        </Grid.Row>
-      </Grid>
-    </div>
-    <Switch>
-      <Route
-        path={`/${ORGS}/${ORG_ID}/load-data/${TELEGRAF_PLUGINS}/${CONTENT_ID}/new`}
-        component={PluginCreateConfigurationWizard}
-      />
-    </Switch>
-  </>
+              <div
+                className="write-data--details-content markdown-format"
+                data-testid="load-data-details-content"
+              >
+                {pageContent}
+              </div>
+            </Grid.Column>
+          </Grid.Row>
+        </Grid>
+      </div>
+      <Switch>
+        <Route
+          path={`/${ORGS}/${ORG_ID}/load-data/${TELEGRAF_PLUGINS}/${CONTENT_ID}/new`}
+          component={PluginCreateConfiguration}
+        />
+        <Route
+          path={`/${ORGS}/${ORG_ID}/load-data/${TELEGRAF_PLUGINS}/${CONTENT_ID}/add`}
+          component={PluginAddToExistingConfiguration}
+        />
+      </Switch>
+    </>
+  )
+}
+
+const mstp = (state: AppState) => ({
+  telegrafs: getAll<Telegraf>(state, ResourceType.Telegrafs),
+})
+
+const mdtp = {
+  fetchTelegrafs: getTelegrafs,
+}
+
```

Comment:
Added this to allow the dropdown to know whether or not to display the "Add to an existing configuration" option. When there are not telegraf configurations, this would not make any sense, and so we hide this option when there are no configurations yet.